src/main/java/ro/utcn/sd/agui/a1/service/QuestionManagementService.java

```diff
@@ -0,0 +1,73 @@
+package ro.utcn.sd.agui.a1.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import ro.utcn.sd.agui.a1.entity.Question;
+import ro.utcn.sd.agui.a1.entity.Tag;
+import ro.utcn.sd.agui.a1.persistence.QuestionRepository;
+import ro.utcn.sd.agui.a1.persistence.RepositoryFactory;
+import ro.utcn.sd.agui.a1.persistence.TagRepository;
+
+import java.sql.Timestamp;
```

Comment:
General remark: if you are writing new code, avoid using the old Timestamp / Date classes, but use the newer LocalDate, LocalDateTime, etc classes from the java.time package.